test.sh

```diff
@@ -104,7 +106,11 @@ With no options passed, runs standard battery of tests (lint, unit, and integrat
     -n, --config-next                     Changes BOULDER_CONFIG_DIR from test/config to test/config-next
     -i, --integration                     Adds integration to the list of tests to run
     -s, --start-py                        Adds start to the list of tests to run
+    -m, --acme                            Adds acme test to the list of tests to run
```

Comment:
We're not interested in having a "python integration tests only" mode. In fact, we're slowly getting rid of all of the python/chisel-based integration tests, moving everything to Go. Don't bother adding this, imo.